src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -0,0 +1,194 @@
+Import-Module Az.KeyVault
+Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.IntegrationAccount -ErrorAction Stop
+
+InModuleScope Arcus.Scripting.IntegrationAccount {
```

Comment:
SO GREAT! Directly integration tests from the start. Great. 👍 
I was working on a separate script so we can re-use some of these Azure connecting functionality.

Reply:
Great, we can perhaps refactor this is the same branch where you're creating the reusable connectivity scripts.
If that's okay with you of course.

Reply:
Yeah, of course! Let me know if you have time here, but I can do a follow-up PR too.